### ISSUE TITLE
Warning: Invalid argument supplied to oneOfType. Expected an array of check functions, but received undefined at index 2.

### ISSUE THREAD

**Issue by Ruchi-29** [carbon-design-system/carbon#6947]
Warning: Invalid argument supplied to oneOfType. Expected an array of check functions, but received undefined at index 2.
Getting 4 errors(Grid- Row and Column, Button ) like this in the console, when integrated carbon-components in my application.
![image](https://user-images.githubusercontent.com/70799769/94798489-48ed7f00-03ff-11eb-818b-cb2755f31a5e.png)

My Button Component is like this:
const ButtonComponent = (props) => {
  let button = props.viewmode === "hl" ? "hl" : "";
  return (
    <div className={props.viewmode === "hl" ? "hl-button-div" : ""}>
      <Button {...props} className={`${button} ${props.className}`} />
    </div>
  );
};

export default ButtonComponent;
How can we overcome this? 
The npm packages I am using .
    "carbon-components": "10.15.0",
    "carbon-components-react": "7.15.0"

**Comment by tw15egan** [comment]
Can you provide a link to a CodeSandbox illustrating the problem so we can help debug what's going on? Thanks!

https://codesandbox.io/s/hungry-darkness-2x51c

**Comment by Ruchi-29** [comment]
It is actually a code in our production. I could probably tell you the approach we took while integrating the carbon components in our application. 
We have been utilizing the components textbox, textarea, numberinput, datalist, dropdown etc. and have wrapped that in our custom component and building a form generator utilizing these components. The button component is not utilized yet we are getting this error warning in the console. I tried upgrading but could not get to a resolution.
When clicking on the error line mentioned it is redirecting to factorywithTypeCheckers.js and button.js prototype error in Button.js
![image](https://user-images.githubusercontent.com/70799769/94891914-f6649f00-04a0-11eb-86c2-b171e553cea6.png)
